ocaml/xcp-rrdd/scripts/rrdd/rrdd.py

```diff
@@ -286,7 +286,7 @@ def wait_until_next_reading(self, neg_shift=1):
                 return
             except socket.error:
                 msg = "Failed to contact xcp-rrdd. Sleeping for 5 seconds .."
-                print >> sys.stderr, msg
+                print(msg, file=sys.stderr)
```

Comment:
Are all callers python3?

Reply:
I guess, but it's not relevant: This line is now unit-tested tested with both 2.7 and 3.11:

I am now enabling flags-based coverage for xen-api on codecov.io, so you should even see the 2.7 and 3.11 coverage on Codecov.io next week:

This "print_function" syntax is natively supported in 2.7 and 3.x.

But thanks to your comment, I also just checked:

Since the `futurize` (which is based on 2to3 code that fails to conver it) commit on
Sep 29 12:12:40 2023, this file has:
```
from __future__ import print_function
```

Which disables the old print statement even for Python2.7, so this line would cause crashes even with Python2 in this file too now:

```py
python2 -c 'from __future__ import print_function;print "hi"'
  File "<string>", line 1
    from __future__ import print_function;print "hi"
```

That means, in any case, it has to be updated, even when used by Python2.7 as well.